desk/app/groups.hoon

```diff
@@ -3360,13 +3345,16 @@
         (go-activity:go-core %role ship roles.u-seat)
       ?:  go-our-host  go-core
       ::
+      =+  was-admin=(go-is-admin our.bowl)
       =.  seats.group
         %-  ~(rep in ships)
         |=  [=ship =_seats.group]
         =+  seat=(~(got by seats) ship)
         =.  seat
           seat(roles (~(uni in roles.seat) roles.u-seat))
         (~(put by seats) ship seat)
+      ?:  !=(was-admin (go-is-admin our.bowl))
+        (go-restart-updates ~)
```

Comment:
You technically only need to do this in the "became an admin" case, right? Not in the "got demoted" one. In the latter all you achieve is dropping admin-only state that you might have learned about, but you can do that locally without touching the subscription if that's necessary. Same applies to all similar instances below.